zephyr/Kconfig

```diff
@@ -31,35 +31,41 @@ config BACDL_ETHERNET
 	help
 	  Enable BACnet Ethernet datalink
 
-config BACDL_MSTP
-	bool "BACnet MSTP datalink"
-	help
-	  Enable BACnet MSTP datalink
-
 config BACDL_ARCNET
 	bool "BACnet ARCNET datalink"
 	help
 	  Enable BACnet ARCNET datalink
 
+config BACDL_MSTP
```

Comment:
What is the benefit of relocating BACDL_MSTP?

Reply:
Rollback